ensime_shared/vim_commands.py

```diff
@@ -0,0 +1,44 @@
+commands = {
```

Comment:
Given the size of the `ensime.py` file I'm not sure how much of a difference it really makes. It's easy to notice that other functions are looking something up from the commands dictionary & that should be the pattern followed.

That being said, if you feel strongly against factoring this out I will put it back.